tests/utf8.rs

```diff
@@ -0,0 +1,10 @@
+extern crate unrar;
+
+use unrar::Archive;
+use std::path::PathBuf;
+
+#[test]
+fn unicode_list() {
+    let mut entries = Archive::new("data/unicode.rar").list().unwrap();
+    assert_eq!(entries.next().unwrap().unwrap().filename, PathBuf::from("te…―st✌"));
```

Comment:
Is this string non-utf8? How would this work? Aren't `&str`s UTF-8? Wouldn't you need to use a `Vec<u8>`?

Reply:
It's UTF-8, I haven't yet come up with any good way of crafting a suitable UTF-16 test case.